### ISSUE TITLE
robo.phar not available on official website

### ISSUE THREAD

**Issue by bedrich-schindler** [consolidation/Robo#660]
Hello,

There is no email contact on your official website robo.li, so I write here. http://robo.li/robo.phar is not available on your webpage. Can you fix this broken link? ... All our Jenkins build tasks fail now :(

Thank you!

**Comment by m4olivei** [comment]
I'm also seeing this on our CI jobs 😞

**Comment by m4olivei** [comment]
Note that per the installation instructions on the http://robo.li/ homepage, you can also download robo from the [Github releases page](https://github.com/consolidation/Robo/releases). For example to download the 1.1.5 release use this link:

https://github.com/consolidation/Robo/releases/download/1.1.5/robo.phar

**Comment by bedrich-schindler** [comment]
@m4olivei I know. We install Robo in Dockerfile but If you wget this GitHub link, you obtain html file because this address is redirected to Amazon. So I temporarily use composer yet, install Robo globally and create symlink for /usr/bin/robo.

**Comment by m4olivei** [comment]
@bedrich-schindler hum, I literally changed our Dockerfile last night to:

```
RUN wget https://github.com/consolidation/Robo/releases/download/1.1.5/robo.phar
````
And it worked. Builds back to normal. Maybe our whether defaults to following redirects. Pretty sure there is an option for that. Anyways glad you found a workaround as well.

**Comment by greg-1-anderson** [comment]
I looked at this briefly, and it seems that robo.phar exists in the commit in the `site` branch used to build the documentation. Perhaps something changed in mkdocs or GitHub pages to prevent / protect against including executables in the documentation.

Rather than fixing this by putting the robo.phar back, my inclination would be to permanently remove the executable from `http://robo.li/robo.phar`, since this address is not encrypted, and the above-mentioned GitHub download is. So, my thought was to dynamically change the link in the documentation to point directly at the GitHub download location in the `release` script, so that users who click on the link on the robo.li site will always go to the https site on GitHub.

The question, then, is what to do for CI users? I don't want to recommend `composer require` (or even [cgr](https://github.com/consolidation/cgr), because that installation method ignores the `composer.lock` file. Adding a redirect from `http://robo.li/robo.phar` to the GitHub download page would be a good idea, but [mkdocs](https://github.com/mkdocs/mkdocs/issues/1102) does not support this yet. Perhaps we could add an html page with a meta-refresh as a post-processing step after mkdocs runs? This is a little inelegant, but it might serve the purpose of CI users if they grabbed the phar via a tool that followed redirects. I suppose another option might be to use some other service than GitHub pages (e.g. [Pantheon](https://pantheon.io)) to host the site. Then we could just put the robo.phar file back and serve it at `https://robo.li/robo.phar`.

**Comment by greg-1-anderson** [comment]
Nevermind the phar build is actually broken. I'll fix it.

**Comment by greg-1-anderson** [comment]
I fixed the build error and released Robo 1.2.1, which replaced the robo.phar at `http://robo.li/robo.phar`.

I would still recommend that CI scripts install from a GitHub release URL with a version, e.g.:
```
https://github.com/consolidation/Robo/releases/download/1.2.1/robo.phar
```
This way, you will continue to get the version of Robo that you initially tested with until you explicitly update to a newer version. New releases should always maintain backwards compatibility, but every now and then a bug slips into a stable release.
